lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -248,7 +248,7 @@ class ToLayoutOpConversionPattern
     // operands.
     for (mlir::Operation *user : op.getResult().getUsers()) {
       if (isa<ttir::Conv2dOp>(user) || isa<ttir::MaxPool2dOp>(user) ||
```

Comment:
Did you try to apply same workaround for all these ops? Were there any issues?

Reply:
I added just Embedding op in this PR so I can write tests, and MaxPool2d op in the previous PR. I would like to separate PRs for core workaround infra from adding workarounds...